source/MRMesh/MRCylinderApproximator.h

```diff
@@ -0,0 +1,413 @@
+#include "MRMeshFwd.h"
+#include "MRLine.h"
+#include "MRVector.h"
+#include "MRMatrix.h"
+#include <Eigen/Eigenvalues>
+#include "MRPch/MRTBB.h"
+#include "MRToFromEigen.h"
+#include "MRConstants.h"
+
+// https://www.geometrictools.com/Documentation/LeastSquaresFitting.pdf 
+
+namespace MR
+{
+
+//using T = float;
```

Comment:
better write some comment about the class

Reply:
done